### ISSUE TITLE
Adding an SPFx Teams tab on the new Teams desktop client is not possible

### ISSUE THREAD

**Issue by canturan** [SharePoint/sp-dev-docs#9253]
### Target SharePoint environment

SharePoint Online

### What SharePoint development model, framework, SDK or API is this about?

💥 SharePoint Framework

### Developer environment

Windows

### What browser(s) / client(s) have you tested

- [ ] 💥 Internet Explorer
- [ ] 💥 Microsoft Edge
- [ ] 💥 Google Chrome
- [ ] 💥 FireFox
- [ ] 💥 Safari
- [ ] mobile (iOS/iPadOS)
- [ ] mobile (Android)
- [ ] not applicable
- [ ] other (enter in the "Additional environment details" area below)

### Additional environment details

New Teams desktop client

### Describe the bug / error

When I add my SPFx app as a new tab to a Team on the new Teams desktop client, the save button stays disabled so that I can't add the app to the Team. 

The same works on the web client and also on the old Teams client without any problem.  

I tried the same with different apps. I could not add any SPFx app as a new tab to teams on the new client.

![image](https://github.com/SharePoint/sp-dev-docs/assets/13345587/8a87fe07-0729-4b1d-897c-4feb86cfe024)

The only error I can see on the console is: 

![image](https://github.com/SharePoint/sp-dev-docs/assets/13345587/082fa09f-85dc-48aa-861f-ed7e60b1e910)

### Steps to reproduce

1. Add a SPFx app as a new tab in Teams 
2. The save button stays disabled

### Expected behavior

The save button becomes enabled (as it always does on the web and old desktop client) so that we can save the config and add the app/tab to the Team.

**Comment by nick-pape** [comment]
Just to clarify here, the Save button is staying disabled on Teams desktop, but this works correctly on Teams web?

**Comment by Undefined-None** [comment]
Same Problem here, but i experienced this behavior starting with spfx 1.16 and the New Teams (startet my testing in late July/ beginning of August 2023). As it was neccessary to use own teams manifest back then (to use with outlook and so on), i thought I do something wrong in the manifests file. Somewhere End of August with spfx 1.17 the teams manifest could be generated automatically by the sharepoint backend again - but the behavioir stayed the same til this day (and spfx 1.18).

On top of this: You can add the Webpart tab in the legacy teams, switch to new teams and it will work BUT you can not open the property pane - it simply does nothing when clicking on "Settings". In SharePoint (and old Teams) everything works as expected.

**Comment by canturan** [comment]
Yes, I have also experienced the same issue with the property pane on the New Teams. It is good point to mention.

**Comment by OliverZeiser** [comment]
I can also confirm this behavior. This needs to be fixed asap as this is a big breaking issue! Otherwise people will switch from SPFx to native teams apps for development and SPFx will lose credibility. Sorry for @mentioning, but we need at least a statement on this @VesaJuvonen @AJIXuMuK

**Comment by AJIXuMuK** [comment]
We're working with Teams partners to figure out the regression and address it.

**Comment by AJIXuMuK** [comment]
The fix has been done and will be rolled out in the next couple of weeks.

**Comment by Undefined-None** [comment]
Thanks for fixing this - i can confirm that i'm able to add spfx apps an configure them.

BUT there is a new issue that is (from my perspective) related to this one: After adding an configuring a Teams-Tab, I unable to alter the configuration, it get's resetted to the initial defaults as soon as i select "Settings" from the newly created Tab. To be more precise: I looks like I end up in the "you added a new Tab-Dialog". Can anyone else confirm this?
